crates/pera-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2395,7 +2395,7 @@ impl AuthorityPerEpochStore {
     /// Important: This function can potentially be called in parallel and you can not rely on order of transactions to perform verification
     /// If this function return an error, transaction is skipped and is not passed to handle_consensus_transaction
     /// This function returns unit error and is responsible for emitting log messages for internal errors
-    fn verify_consensus_transaction(
+    async fn verify_consensus_transaction(
```

Comment:
Changed this function to be async so I'll be able to acquire the `dwallet_mpc_manager` lock